samples/sampler/stories/default/QuickGuidePanels.js

```diff
@@ -13,31 +19,72 @@ export default {
 	component: 'QuickGuidePanels'
 };
 
-export const _QuickGuidePanels = (args) => (
-	<QuickGuidePanels
-		current={args['current']}
-		nextButtonVisibility={args['nextButtonVisibility']}
-		onBack={action('onBack')}
-		onChange={action('onChange')}
-		onClose={action('onClose')}
-		onNextClick={action('onNextClick')}
-		onPrevClick={action('onPrevClick')}
-		prevButtonVisibility={args['prevButtonVisibility']}
-		total={args['total']}
-	>
-		<QuickGuidePanels.Panel>
-			This is Panel 0
-		</QuickGuidePanels.Panel>
-		<QuickGuidePanels.Panel>
-			<div style={{height: '100%', width: '100%', backgroundColor: 'grey'}}>
-				This is Panel 1
-			</div>
-		</QuickGuidePanels.Panel>
-		<QuickGuidePanels.Panel aria-label={'aria test'}>
-			This is Panel 2
-		</QuickGuidePanels.Panel>
-	</QuickGuidePanels>
-);
+export const _QuickGuidePanels = (args) => {
+	const screenWidth = window?.screen.width;
+	const screenHeight = window?.screen.height;
+	const widthRatio = screenWidth / 1920;
+	const heightRatio = screenHeight / 1080;
+
+	return (
+		<>
+			<Panel css={css}>
+				<Button css={css} focusEffect="static" size="small">Button</Button>
+				<br />
+				<SwitchItem inline>item</SwitchItem>
+			</Panel>
+			<Popup
+				open
+				position="fullscreen"
+				scrimType="transparent"
+				style={{backgroundColor: 'transparent'}}
```

Comment:
We could.... use public css with `applySkins` mixin to override the background color. But I'm ok with this too.

Reply:
I fixed to use public css.